.github/workflows/snorkell-auto-documentation.yml

```
@@ -0,0 +1,19 @@
+  workflow_dispatch:
+
+jobs:
+  Documentation:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Penify DocGen Client
+      uses: SingularityX-ai/snorkell-documentation-client@v1.0.0
+      with:
+        client_id: ${{ secrets.SNORKELL_CLIENT_ID }}
+        api_key: ${{ secrets.SNORKELL_API_KEY }}
+        branch_name: "master"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)